salt/srv/salt/_modules/smart.py

```diff
@@ -0,0 +1,108 @@
+from subprocess import PIPE, Popen
+import salt.config
+import salt.loader
+import logging
+
+log = logging.getLogger(__name__)
+__opts__ = salt.config.minion_config('/etc/salt/minion')
+__grains__ = salt.loader.grains(__opts__)
+
+ARECA = 'areca'
+MDADM = 'mdadm'
+
+
+def _get_raid_devices():
```

Comment:
dangerously misleading method name -- you're piping lspci into "head -1" so arbitrarily picking the first raid controller (sorted by PCI bus ID)